camel-idea-plugin/build.gradle

```diff
@@ -23,7 +23,7 @@ apply plugin: 'java'
 apply plugin: 'org.jetbrains.intellij'
 
 intellij {
-    version = '2019.3.1'
+    version = '2019.3'
```

Comment:
I am not quite sure 🙂 I hope that it will just expand the plugin availability to 2019.3.

It does work when I install the plugin via zip file, for any version (2019.3 or 3.1)

Reply:
okay. I will need to do a little more testing and investigation before I can approve it